src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User user = new User(email, password);
+        if (findByEmail(email) == null) {
+            return false;
+        }
+        if (user.getEmail().equals(findByEmail(email).getEmail())
+                && user.getPassword() == findByEmail(email).getPassword()) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
let's create `UserService` object, so we can call `findByEmail()` on that object
let's save result of `findByEmail()`, so we don't call method each time we want to use it's result
use && operator directly in return because && expression returns boolean